libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/BaseInvokeDialog.cs

```diff
@@ -36,7 +38,7 @@ public BaseInvokeDialog(string dialogIdToCall = null, IDictionary<string, string
         /// Configurable options for the dialog. 
         /// </value>
         [JsonProperty("options")]
-        public object Options { get; set; } = new JObject();
+        public object Options { get; set; }
```

Comment:
>object [](start = 15, length = 6)

It is not clear to me if this should be an ObjectExpression....thoughts?

Reply:
Hmm.  What options are allowed?  I suspect this OK.

---
In reply to: [367070814](https://github.com/microsoft/botbuilder-dotnet/pull/3253#discussion_r367070814) [](ancestors = 367070814)